src/Calendar.js

```diff
@@ -55,26 +54,83 @@ const weekNumberCalculation = date => {
   return end - start + 1 // 0 weeks after start is week 1
 }
 
-export default forwardRef(({ state }, ref) => {
-  const customEvents = {
-    eventContent: e => formatEventContent(state)(e),
-  }
-
+export default function Calendar({ timetableState }) {
   // Set the initial date to max(start of sem, today)
   const startOfSemester = getStartOfSession()
   const initialDate =
     startOfSemester && startOfSemester.getTime() > new Date().getTime()
       ? startOfSemester
       : new Date();
 
+  // Where the events are stored
+  const [calendarEvents, setCalendarEvents] = useState([])
+
+  useEffect(() => {
```

Comment:
Has this been re-added? What happens if you don't use a useEffect?

Reply:
I initially put it in to reduce re-renders from a different bug that was solved.